src/components/Card.js

```diff
@@ -5,23 +5,49 @@ import { removeBooksAsync } from '../redux/books/books';
 
 const Card = (props) => {
   const { book } = props;
-  console.log(book);
   const dispatchRemove = useDispatch();
   const handleRemoveBook = () => {
     removeBooksAsync(book.id)(dispatchRemove);
   };
 
   return (
-    <li className="bookInfo">
-      <p>{book.name}</p>
-      <p>{book.author}</p>
-      <p>{book.category}</p>
-      <p>{book.id}</p>
-
-      <button type="button" onClick={handleRemoveBook}>
-        Remove Book
-      </button>
-    </li>
+    <div className="card-complete">
+      <div className="info">
+        <li className="bookInfo">
+          <p className="book-category">{book.category}</p>
+          <h1 className="book-title">{book.name}</h1>
+          <p className="book-author">{book.author}</p>
+        </li>
+        <div className="buttons">
+          <button type="button" className="info-btn">Comments</button>
+          <button type="button" onClick={handleRemoveBook} className="info-btn">
+            Remove
+          </button>
+          <button type="button" className="info-btn">Edit</button>
+        </div>
+      </div>
+      <div className="left d-flex">
+        <div className="porcentage">
+          <div className="wrapper" data-anim="base wrapper">
+            <div className="circle" data-anim="base left" />
+            <div className="circle" data-anim="base right" />
+          </div>
+          <div className="presentage d-flex">
+            <span className="pres">
+              100%
+            </span>
+            <span className="completed">
+              Completed
+            </span>
+          </div>
+        </div>
+      </div>
+      <div className="progress">
+        <h4>CURRENT CHAPTER</h4>
+        <span className="lesson">Chapter 17</span>
+        <button className="Remove" type="button">Update progress</button>
+      </div>
+    </div>
```

Comment:
- [ ] Kindly make the border of the general container vertically align with that of the _**ADD NEW BOOK**_ header for the form control.

### Your Design

![alignment](https://user-images.githubusercontent.com/77106962/161406924-db593c60-596d-4929-b114-2bafb11b63e8.png)

### Zeplin's Template

![image](https://user-images.githubusercontent.com/77106962/161406979-15646b31-7255-45ea-9439-55c127ee8791.png)